app/src/main/java/com/example/parkingsystem_kotlin/mvp/view/ParkingReservationActivityView.kt

```diff
@@ -0,0 +1,43 @@
+package com.example.parkingsystem_kotlin.mvp.view
+
+import android.app.Activity
+import android.widget.EditText
+import android.widget.Toast
+import com.example.parkingsystem_kotlin.databinding.ActivityParkingReservationBinding
+import com.example.parkingsystem_kotlin.mvp.contract.ParkingReservationActivityContract
+import com.example.parkingsystem_kotlin.mvp.view.base.ActivityView
+import com.example.parkingsystem_kotlin.util.DateUtils.showDatePickerDialog
+
+class ParkingReservationActivityView(private val binding: ActivityParkingReservationBinding, activity: Activity) :
+    ParkingReservationActivityContract.View, ActivityView(activity) {
+
+    override fun showToastOfInput(input: Int) {
+        context?.let {
+            Toast.makeText(it, input.toString(), Toast.LENGTH_SHORT).show()
+        }
+    }
+
+    override fun getCodeFromInput(): String {
+        return binding.inputNumberParkingCode.text.toString()
+    }
+
+    override fun getSlotFromInput(): String {
+        return binding.inputNumberParkingSlot.text.toString()
```

Comment:
instead of return here you can use String = binding...

Reply:
Thanks